akka-typed/src/main/scala/akka/typed/cluster/ActorRefResolver.scala

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed.cluster
+
+import akka.typed.ActorSystem
```

Comment:
@johanandren I changed this to use the typed extension instead, feels more future proof.